ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.6.yaml

```diff
@@ -51,6 +51,12 @@ tests:
     test:
     - ref: windows-e2e-operator-test-with-custom-vxlan-port
     workflow: ipi-aws-ovn-hybrid-custom-vxlan-port
+- as: vsphere-e2e-vsphere
```

Comment:
Shouldn't you be replacing the `aws-e2e-operator-with-custom-vxlan-port` with this test? Or do you want to do that only after https://github.com/openshift/windows-machine-config-operator/pull/139 merges.